wording

bstamour opened the pull request.

Commit:
Changed static lambdas to member funcs to grab Self within
struct default-state-impls

Commit:
Checkpoint for the mailing deadline. Plan is to continue to work on the
other adaptors/factories for the meeting.

Commit:
wording fleshed out. surely there are issues, but it's a start.

Commit:
run_loop

some cleanup

Commit:
new formulation: just and friends

Commit:
then and friends

Commit:
let_value (mostly)

Commit:
cleanup the old sections

lewissbaker commented on isocpp/inline_op_states.org:
```diff
@@ -66,8 +66,8 @@
-| Date     | *2024-11-15*                          |
+| Document | *P3424R2*                             |
+| Date     | *2025-12-10*                          |
 | Reply To | *Lewis Baker <lewissbaker@gmail.com>* |
```
Need to add @bstamour as an author.

lewissbaker commented on isocpp/inline_op_states.org:
```diff
@@ -66,8 +66,8 @@
-| Document | *D3424R1*                             |
-| Date     | *2024-11-15*                          |
+| Document | *P3424R2*                             |
+| Date     | *2025-12-10*                          |
```
Date is (very far) in the future ;)

Commit:
bulk

lewissbaker committed:
Updated wording for [exec.snd.expos]

Define/update the new helper facilities for child operation states
and remove old facilities no longer needed.

Fix the formatting of some of the existing wording sections
(add paragraph numbers, markup exposition only symbols, etc.).

lewissbaker committed:
Update file header information.

Fix date, add Bryan as co-author.

lewissbaker committed:
Fix up the wording for [exec.just].

Add in more context so that paragraph numbering works properly.
Change definition of `basic-operation` to use data-member rather than
base-class for the tuple of values.
Fix formatting.

lewissbaker committed:
Update wording of [exec.read.env].

Also update the primary template for `sender-impls-for` to actually
have a definition that inherits from the `default-sender-impls` so
that we don't need to define it for senders that don't modify
get-attrs.

lewissbaker committed:
Update wording for [exec.then].

Fixed some formatting and some exposition-only identifiers.
Remove the need to specialize sender-impls-for as we are just
handwaving generically for the completion-signatures calculations in
`basic-sender::get_completion_signatures()` and so don't need to worry
about specializing sender-impls-for to handle that.

Commit:
minor cleanups
